relayer/src/services/boba.ts

```diff
@@ -0,0 +1,95 @@
+import { CrossChainMessenger, MessageStatus, MessageReceiptStatus } from "@eth-optimism/sdk-1.0.2";
```

Comment:
Yes, but we have a problem due to the different versions of optimism sdk being used in these 2 files. 

In the `BobaRelayerService` we have to use `optimism/sdk@1.0.2` since this SDK version provides us with the correct API/types for communication with the boba network contracts. Boba haven't upgraded their network to use/fork the latest features implemented on Optimism therefore we cannot use the latest version of optimism SDK for their network.)

Unlike the boba relayer the `OptimismRelayerService` is using `optimism/sdk@latest` (obviously) which has the up-to-date API we need for communicating with the Optimism specific contracts responsible for the message bridging.